tests/Core/Ruleset/RuleInclusionTest.xml

```diff
@@ -31,7 +31,7 @@
 	    </properties>
 	</rule>
 
-	<rule ref="./../PHP_CodeSniffer/src/Standards/Generic/Sniffs/NamingConventions/CamelCapsFunctionNameSniff.php">
+	<rule ref="./src/Standards/Generic/Sniffs/NamingConventions/CamelCapsFunctionNameSniff.php">
```

Comment:
You are breaking the test. The whole point of the test is that path resolution is checked and as this is a test file, it is save to presume the path.

Suggest: close.

Reply:
Sorry, but `PHP_CodeSniffer` only exists if you clone  the git repo under this name... and not from the pear archive

Reply:
btw, pr#2847 will skip it